parastell/invessel_build.py

```diff
@@ -352,15 +352,18 @@ def export_cad_to_dagmc(self, filename='dagmc', export_dir=''):
 
         model = cad_to_dagmc.CadToDagmc()
 
-        for name, component in self.Components.items():
-            model.add_cadquery_object(
-                component, material_tags=[self.radial_build[name]['mat_tag]']]
-            )
+        for component in self.Components.values():
+            model.add_cadquery_object(component)
+        
+        material_tags = [
+            component['mat_tag'] for component in self.radial_build.values()
+        ]
```

Comment:
Do the relative orders of these things matter?  One loops over `self.Components` and one over `self.radial_build`.  Presumably those things are both built in the same order, but nothing guarantees it...?